src/i18n/lang/nl.json

```diff
@@ -5,11 +5,8 @@
             "filtersTitle": "Filters",
             "priorityLabel": "Prioriteit",
             "statusLabel": "Status",
-            "startDatePlaceholder": "Begindatum",
-            "startDateError": "Ongeldige datum",
+            "datePlaceholder": "Aanmaakdatum",
             "startDateExample": "(Bijv: 28/04/2022)",
```

Comment:
It looks like your forgot to remove "startDateExample" and "endDateExample" here (**nl.json**) as well as in **no.json**.